scene/3d/skeleton_3d.cpp

```diff
@@ -938,14 +1054,14 @@ void Skeleton3D::force_update_bone_children_transforms(int p_bone_idx) {
 	ERR_FAIL_INDEX(p_bone_idx, bone_size);
 
 	Bone *bonesptr = bones.ptrw();
-	thread_local LocalVector<int> bones_to_process;
-	bones_to_process.clear();
-	bones_to_process.push_back(p_bone_idx);
 
-	uint32_t index = 0;
-	while (index < bones_to_process.size()) {
-		int current_bone_idx = bones_to_process[index];
+	// Loop through nested set.
+	for (int offset = 0; offset < bone_size; offset++) {
+		if (!bone_global_pose_dirty[offset]) {
```

Comment:
It would be better to use pointers for `bone_global_pose_dirty` and `nested_set_offset_to_bone_index` instead of `operator []` as done with `bonesptr`.

Reply:
I'm not sure if there's much of a difference for `LocalVector`. I agree that using pointers is probably a tiny bit faster, but I'd rather keep it as it is than adding new variables. I think it's more readable that way.

Reply:
If necessary, it could be followed up later as a PR for performance improvement.